CHANGELOG.md

```diff
@@ -1,9 +1,39 @@
 # Change log for Microsoft365DSC
 
+1.21.1006.1
+
+* BREAKING CHANGES
+
+* AADApplication
+  * Removed support for the Oauth2AllowImplicitFlow, SamlMetadataUrl and
+    Oauth2AllowUrlPathMatching properties;
+* AADMSGroup
+  * Renamed resource to AADGroup;
+* AADMSGroupLifecyclePolicy
+  * Renamed resource to AADGroupLifecyclePolicy;
+* AADPolicy
+  * Replaced resource by the new AADTokenLifetimePolicy one;
+* DEPENDENCIES
+  * Added dependency on Microsoft.Graph.Applications;
+  * Added dependency on Microsoft.Graph.Groups;
+  * Added dependency on Microsoft.Graph.Identity.DirectoryManagement;
+  * Added dependency on Microsoft.Graph.Identity.SignIns;
+  * Removing dependency on AzureADPreview;
+  * Updated Microsoft.PowerApps.Administration.PowerShell to version 2.0.137
+* MISC
+  * Renamed the GlobalAdminAccount parameter to Credential across all resources;
+  * Revamped entire Azure AD Workload to leverage Microsoft Graph;
+  * Doing an unattended export (without the -Quiet) will always launch the
```

Comment:
In the code the Quiet parameter is renamed to LaunchWebUI.....sounds like this statement isn't true. You have to specify that parameter to launch the webUI, right?

Reply:
Rephrased entry